### PR TITLE
#1288 Add interface for slack node

### LINKED ISSUE #1288: Add get slack node API in data interface
In prevision of IIDM library 1.3 and to finish DFL dev, data interface needs a getter for this data,if present. For now, this getter will be empty

### PR DESCRIPTION
**Pull request by lecourtoisflo**
closes #1288

### CODE REVIEW

File: dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp [hunk 1]
```diff
@@ -1325,6 +1325,12 @@ DataInterfaceIIDM::getStaticParameterBoolValue(const std::string& staticID, cons
   return findComponent(staticID)->getStaticParameterValue<bool>(refOrigName);
 }
 
+boost::optional<std::string>
+DataInterfaceIIDM::getSlackNodeBusId() const {
```
Comment (rosiereflo) [inline dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp:1329]: Shouldn't this go into the network API?

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp (hunk 1 above)
Comment (lecourtoisflo) [reply dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp:1329]: For me it was a global API for data interface to be the most generic possible. Should it be in NetworkInterface ?

Comment (rosiereflo) [reply dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp:1329]: @marcochiaramello any opinion?

Comment (marcochiaramello) [reply dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp:1329]: I think the concept of slacknode is useful regardless the data format, so I'd prefer it in NetworkInterface

**Review by rosiereflo — APPROVED** [review]

**Review by marcochiaramello — APPROVED** [review]

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**